spec/adaptors/netsuite/department/operations/create_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+support :input_helpers,
+        :operation_shared_examples,
+        :netsuite_helpers
+
+RSpec.describe LedgerSync::Adaptors::NetSuite::Department::Operations::Create do
+  include InputHelpers
+  include NetSuiteHelpers
+
+  let(:resource) { LedgerSync::Department.new(department_resource) }
```

Comment:
I prefer using `FactoryBot.create(:department)`, as this is the old way we used to create resources.